src/Licenses/Entities/License/License.php

```diff
@@ -317,6 +319,11 @@ class License extends AbstractEntity
      */
     private $warnings;
 
+    /**
+     * @var Security
+     */
+    private $security;
```

Comment:
```suggestion
    private Security $security;
```

Reply:
Type hinting on class properties is supported from php7.4+. however, this repo is compatible with >= php7.2

Reply:
Right :+1: